halibot/halibot.py

```diff
@@ -44,6 +42,8 @@ def __init__(self, **kwargs):
 		self.use_config = kwargs.get("use_config", True)
 		self.use_auth = kwargs.get("use_auth", True)
 		self.workdir = kwargs.get("workdir", ".")
+		self.config = kwargs.get("config", {})
```

Comment:
Issue here: When we load a config from a file via `._load_config`, we update `halibot.packages.__path__`. If a config is supplied, that step is skipped. Will fix in next edit as well, just need to check if we are supplied a config and update the path accordingly.